### PR TITLE
Language Support for ATS

### PR DESCRIPTION
**Pull request by pykenny**
## Purpose
Adding language support and icon art for [ATS](http://www.ats-lang.org/).

Language support at `tokei` is still undergoing review, and will provide screenshot once it's available.

## Logo Reference
https://www.cs.bu.edu/~hwxi/atslangweb/MYDATA/theLogo.png

## Preview
|view|
| - |
|</img>|

## Related PR/issues:
 - #490
 - XAMPPRocky/tokei#840

### CODE REVIEW

**Comment by spenserblack** [comment]
I wonder when there will be a new release of tokei. https://github.com/XAMPPRocky/tokei/pull/840 was merged a long time ago.

**Comment by o2sh** [comment]
Yeah, tokei's last [release](https://github.com/XAMPPRocky/tokei/releases/tag/v12.1.2) is more than year old 😟

**Comment by o2sh** [comment]
This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.

File: languages.yaml [hunk 1]
```diff
@@ -104,6 +104,39 @@ Assembly:
...
+  colors:
+    ansi:
+      - yellow
+      - red
+      - blue
+    hex:
+      - #FFFF00
+      - #FF0000
+      - #0000FF
+    chip: #1AC620
```
Comment (spenserblack) [inline languages.yaml:138]:
```suggestion
      - '#FFFF00'
      - '#FF0000'
      - '#0000FF'
    chip: '#1AC620'
```

**Review by spenserblack — CHANGES_REQUESTED** [review]

**Review by spenserblack — APPROVED** [review]

**Comment by spenserblack** [comment]
Wow, every day I find a new pending PR :laughing:

@pykenny FYI: #1152

**Comment by pykenny** [comment]
@spenserblack Sorry for not noticing conflict with upstream. Try the CI pipeline again?

**Comment by spenserblack** [comment]
@pykenny no worries. Although the CI will likely still fail right now since tokei hasn't been released with support for ATS yet. Though hopefully gengo will fix that :wink:

Anyway, I don't think you have to worry about this PR. You've done all that we could ask a contributor to do to add a new language. it just depends on us fixing upstream issues.

**Comment by pykenny** [comment]
There's some major update on onefetch, so redo the commit from current main branch for simplicity.

File: languages.yaml [hunk 2]
```diff
@@ -138,6 +138,39 @@ Assembly:
...
+    ansi:
+      - yellow
+      - red
+      - blue
+    hex:
+      - "#FFFF00"
+      - "#FF0000"
+      - "#0000FF"
+    chip: "#1AC620"
+  serialization: ATS
```
Comment (spenserblack) [inline languages.yaml:173]:
```suggestion
```
Our serialization is typically lowercase. This populates the available options for the `--ascii-language` option. You don't really need to use this option unless you're adding special characters (`CSharp` -> `c#`) or renaming (`Sh` -> `shell`).

**Review by spenserblack — CHANGES_REQUESTED** [review]

**Pull request merged**